core/src/avm2/globals/README.md

```diff
@@ -1,7 +1,7 @@
-# Ruffle AVM2 globals
+# AVM2 globals
 
-This directory contains implementations of Flash global
-definitions (e.g. `Object`, `flash.geom.Point`, `flash.utils.getDefinitionByName`)
+This directory contains implementations of global classes
```

Comment:
Freestanding functions aren't classes, so I think this should still say 'definitions'